src/views/product-landing/components/themed-image/themed-image.module.css

```diff
@@ -0,0 +1,12 @@
+.root {
+	align-items: center;
+	display: flex;
+	width: 100%;
```

Comment:
The widths of the images are all different, so we need to use the `fill` approach and set the height here.